proto/nftmarket/tx.proto

```diff
@@ -12,17 +12,13 @@ service Msg {
   rpc MintNft(MsgMintNft) returns (MsgMintNftResponse);
   rpc ListNft(MsgListNft) returns (MsgListNftResponse);
   rpc CancelNftListing(MsgCancelNftListing) returns (MsgCancelNftListingResponse);
-  rpc ExpandListingPeriod(MsgExpandListingPeriod) returns (MsgExpandListingPeriodResponse);
   rpc PlaceBid(MsgPlaceBid) returns (MsgPlaceBidResponse);
   rpc CancelBid(MsgCancelBid) returns (MsgCancelBidResponse);
   rpc SellingDecision(MsgSellingDecision) returns (MsgSellingDecisionResponse);
   rpc EndNftListing(MsgEndNftListing) returns (MsgEndNftListingResponse);
   rpc PayFullBid(MsgPayFullBid) returns (MsgPayFullBidResponse);
   rpc Borrow(MsgBorrow) returns (MsgBorrowResponse);
   rpc Repay(MsgRepay) returns (MsgRepayResponse);
-  rpc MintStableCoin(MsgMintStableCoin) returns (MsgMintStableCoinResponse);
-  rpc BurnStableCoin(MsgBurnStableCoin) returns (MsgBurnStableCoinResponse);
-  rpc Liquidate(MsgLiquidate) returns (MsgLiquidateResponse);
```

Comment:
Removed ExpandListingPeriod, MintStableCoin, BurnStableCoin, Liquidate messages - can you confirm the changes ? @mkXultra